### PR TITLE
Replace hawkular with middleware manager

### PR DESCRIPTION
**Pull request by rubenvp8510**
This PR replace all display name and descriptions of the template to middleware manager, also, now is using service signed certificates for SSL communication.

### CODE REVIEW

**Comment by theute** [comment]
Few comments:
- Templates need to be updates in 1.5 (which is really 3.5), 3.6 and 3.7 folders
- We should have the ephemeral one as well
-  It should be "Middleware Manager" and not "Middleware Manager Services" everywhere. I think we can replace "Hawkular" with "Hawkular Services" everywhere upstream and just replace "Hawkular Services" with "Middleware Manager" in our script.

File: roles/openshift_examples/files/examples/v3.6/cfme-templates/jboss-middleware-manager-template.yaml [hunk 1]
```diff
@@ -175,6 +190,9 @@ objects:
         - name: h-services-data
           persistentVolumeClaim:
             claimName: h-services-pvc
+        - name: hawkular-secret-volume
+          secret:
+            secretName: hawkular-ssl-secret
```
Comment (hhovsepy) [inline roles/openshift_examples/files/examples/v3.6/cfme-templates/jboss-middleware-manager-template.yaml]: Will there be the SSL solution with secrets, or there should be Routes solution?

Comment (rubenvp8510) [reply roles/openshift_examples/files/examples/v3.6/cfme-templates/jboss-middleware-manager-template.yaml]: It's a routes solution, but even that solution needs for mount some secret, in this case is the secret created by openshift signed service.

**Comment by theute** [comment]
@rubenvp8510 is it ok to merge now ?

**Comment by rubenvp8510** [comment]
@theute Yes, this is ok to merge, the only last thing we need to do in order to have SSL correct working on this, is to update the docker images. but if the images are not updated it should work on the old way. (using self-signed certificate)

**Comment by hhovsepy** [comment]
have tested templates with the newest images we have, all worked

**Comment by theute** [comment]
@sdodson Could you please merge this one ?
Also I realize that there is a new v3.8 since 4 days, that directory would need to be in sync as well...

**Comment by mwringe** [comment]
@ewolinetz is there anything else we need to do here to get this merged?

**Comment by mwringe** [comment]
@ewolinetz can we please get this merged or know what we need to do to get this pushed through?

**Comment by rubenvp8510** [comment]
@ewolinetz I added one small commit to this PR.

**Comment by ewolinetz** [comment]
@rubenvp8510 ack, would you mind squashing these commits into a single one?

**Comment by rubenvp8510** [comment]
@ewolinetz done! :)

**Comment by rubenvp8510** [comment]
@ewolinetz any idea why this is failing?

**Comment by ewolinetz** [comment]
@rubenvp8510 may just be a flake
/retest

**Comment by ewolinetz** [comment]
bot, retest this please

**Pull request closed without merging**